scenarioo-client/app/search/search.html

```diff
@@ -25,7 +25,7 @@
      data-treedata="vm.results.resultSet.children"
      data-treemodel="vm.treemodel"
      data-click-action="vm.goToRelatedView(node)"
-     data-first-column-title="Search Results"
+     data-first-column-title="Search Results (showing {{vm.hits}} of {{vm.totalHits}} hits)"
```

Comment:
Since we do not have a real paging yet, does displaying "hits of totalHits" make sense at all. Or would we see something like "200 of 15642 hits" in case there are more hits ??